src/containers/ProfileDetails/components/CaptivePortal/index.js

```diff
@@ -451,7 +451,7 @@ const CaptivePortalForm = ({
             ]}
           >
             <div className={styles.InlineDiv}>
-              <Input className={globalStyles.field} placeholder="http://... or https://..." />
+              <Input className={globalStyles.field} placeholder="http://... or https://..." defaultValue={details.externalCaptivePortalURL}/>
```

Comment:
this isn't the solution, the issue is because it's inside a div needs to be a direct child of Item